evolvedbinary/prosemirror-lwdita
# Unit tests for the backend API

This PR adds tests for the GitHub router and the GitHub controller.

# how to test this PR
* checkout the branch
* install new dependencies `yarn instal`
* run the tests `yarn run test:backend`
* check the backend test on CI
